tests/visualize_test.py

```diff
@@ -172,15 +174,21 @@ def long_cvt_archive():
 @pytest.fixture(scope="module")
 def sliding_archive():
     """Deterministically created SlidingBoundariesArchive."""
-    archive = SlidingBoundariesArchive(2, [10, 20], [(-1, 1), (-1, 1)], seed=42)
+    archive = SlidingBoundariesArchive(solution_dim=2,
+                                        dims=[10, 20],
+                                        ranges=[(-1, 1), (-1, 1)],
+                                        seed=42)
     add_random_sphere(archive, (-1, 1), (-1, 1))
     return archive
 
 
 @pytest.fixture(scope="module")
 def long_sliding_archive():
     """Same as above, but the behavior space is longer in one direction."""
-    archive = SlidingBoundariesArchive(2, [10, 20], [(-2, 2), (-1, 1)], seed=42)
+    archive = SlidingBoundariesArchive(solution_dim=2,
+                                        dims=[10, 20],
```

Comment:
Did you run `yapf`? `dims`, `ranges`, and `seed` should line up with `solution_dim` -- same with line 278 below.